.circleci/config.yml

```diff
@@ -13,18 +13,18 @@ executors:
   clojure:
     working_directory: /home/circleci/metabase/metabase/
     docker:
-      - image: circleci/clojure:lein-2.8.1
+      - image: circleci/clojure:lein-2.9.3
```

Comment:
It's not, but I realized it was available so I went ahead and upgraded while I was poking at everything else

Reply:
I'd vote for making each change focused on the concept it's introducing. right now it'd be hard for us to disentangle effects from this upgrade vs changes in this CI build config, especially since lein is a build tool...